src/Rule/IsIPAddress.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+
+namespace Morebec\Validator\Rule;
+
+
+use Morebec\Validator\ValidationRuleInterface;
+
+/**
+ * Validates that a value is an Ip Address
+ */
+class IsIPAddress implements ValidationRuleInterface
+{
+    /**
+     * @var string
```

Comment:
Can you set the docstring to `string|null` please?

Reply:
Fixed docstring
https://github.com/Morebec/Validator/pull/14/commits/90204edf801163c8c75acda36ef702be6c49e705